x-pack/legacy/plugins/canvas/i18n/components.ts

```diff
@@ -340,6 +340,50 @@ export const ComponentStrings = {
         defaultMessage: 'Select an element to show expression input',
       }),
   },
+  ExpressionInput: {
+    getArgReferenceAliasesDetail: (aliases: string) =>
+      i18n.translate('xpack.canvas.expressionInput.argReferenceAliasesDetail', {
+        defaultMessage: '**Aliases**: {aliases}',
```

Comment:
Will the translator know to keep the `**`s, or should those be extracted into `values`?

Reply:
That's an excellent point. I'll ask the team in #kibana-localization